odc/dscache/tools/tiling.py

```diff
@@ -40,6 +40,21 @@
         )
         for n in (10, 20, 30, 60)
     },
+    "au_extended": GridSpec(
+        crs=epsg3577,
+        tile_size=(96_000.0, 96_000.0),
+        resolution=(-96_000, 96_000),
+        origin=(-4416000.0, -6912000.0),
```

Comment:
GridSpec origins are Y, X, so I think this needs to be reversed:

```suggestion
        origin=(-6912000.0, -4416000.0),
```